cmux-tui/scripts/smoke-windows-conpty-resize.py

```
@@ -69,29 +82,157 @@ def wait_for_socket(
+        # Before startup completes, backpressure preserves every chunk. Once
+        # the marker has been observed, later chunks go to bounded tail
+        # diagnostics instead of an unbounded queue. stop_event releases a
+        # producer blocked on a full queue during cleanup.
+        if self.startup_complete.is_set():
+            if isinstance(event, str):
+                self._append_tail(event)
+            return
+        while not self.stop_event.is_set() and not self.startup_complete.is_set():
+            try:
+                self.events.put(event, timeout=0.05)
+                return
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)